src/sos/task_executor.py

```diff
@@ -401,7 +401,7 @@ def execute_master_task(self, task_id, params, master_runtime, sig_content):
         # node. The "n_procs" is the "number of cpus per job per node", which,
         # is "cores" (per task) * "n_workers" (per node). Therefore,
         # "n_procs" should not be used.
-        n_nodes, n_procs = self._parse_num_workers(env.config["worker_procs"])
+        n_nodes, _ = self._parse_num_workers(env.config["worker_procs"])[0]
```

Comment:
What are you doing here?

Reply:
_ is not needed if you add [0].

Reply:
I just changed n_nodes, _   self._parse_num_workers(env.config["worker_procs"])[0] to n_nodes, _   self._parse_num_workers(env.config["worker_procs"]) and I pushed the changes but I do not see them on GitHub.

Reply:
check if you are on the right branch.